src/main/java/emu/grasscutter/utils/StartupArguments.java

```diff
@@ -36,10 +37,7 @@ private StartupArguments() {
         },
 
         // Aliases.
-        "-v", StartupArguments::printVersion,
-        "-debugall", parameter -> {
```

Comment:
Should make `-debugall` override the config with roughly the same settings as it produced before, rather than removing it.

Reply:
Code changed to override the config with "-debugall"